app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,98 @@
+class ReviewsController < ApplicationController
+  before_action :set_review, only: [:show, :edit, :update, :destroy]
+  # GET /reviews
+  # GET /reviews.json
+  def index
+    @reviews = Review.all
+  end
+
+  # GET /reviews/1
+  # GET /reviews/1.json
+  def show
+  end
+
+  # GET /reviews/new
+  def new
+    @restaurant_id = params[:restaurant_id]
```

Comment:
Not something I want you to do now, but a suggestion:
you know what could have been cooler - if you passed the entire restaurant and not just the id. This way you could put a title like "New review for Falafel Shkuna".